### PR TITLE
Efficiently Insert Multiple Rows

### PR DESCRIPTION
**Pull request by amirsarb**

### CODE REVIEW

**Review by theangrydev — APPROVED** [review]

File: SQL-Insert-Multiple-Rows/insert-multiple-rows.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+INSERT INTO Student (id, name, national_id)
+VALUES (1001, 'John Liu', '123345566'),
+       (1003, 'Rita Ora', '132345166'),
+       (1007, 'Philip Lose', '321345566'),
+       (1010, 'Samantha Prabhu', '3217165566');
+
+INSERT INTO Student (id, name, national_id)
+SELECT id, name, national_id
+FROM old_student
```
Comment (theangrydev) [inline SQL-Insert-Multiple-Rows/insert-multiple-rows.sql:17]:
```suggestion
FROM Old_student
```

**Review by theangrydev — CHANGES_REQUESTED** [review]

**Review by theangrydev — APPROVED** [review]

**Review by amirsarb — COMMENTED** [review]
Capital fixed.

**Comment by amirsarb** [comment]
Folder changed

**Review by theangrydev — APPROVED** [review]

**Review by amirsarb — COMMENTED** [review]
### Note on Section 6 (Using UNION ALL)

The method described in Section 6 (Using UNION ALL) is PostgreSQL-specific. It is not supported in SQL Server and MySQL. 

Please review and confirm that this limitation is clearly communicated in the tutorial.

Thank you!

**Review by amirsarb — COMMENTED** [review]
change

**Review by amirsarb — COMMENTED** [review]
change

**Pull request closed without merging**